src/databases/hashmap_db.rs

```diff
@@ -1,6 +1,16 @@
+#[cfg(not(feature = "std"))]
+use alloc::{
+    fmt,
+    fmt::Display,
```

Comment:
fmt is present in `core`. It can be imported from the same place for both

Reply:
Same with dispaly

Reply:
done